cardano_node_tests/tests/tests_plutus/test_delegation.py

```diff
@@ -829,9 +833,9 @@ def test_delegate_deregister(  # noqa: C901
         reward_error = ""
 
         # To speed up test run, skip waiting for rewards in selected scenarios
-        skip_rewards_check = (use_reference_script and not use_build_cmd) or (
-            not use_reference_script and use_build_cmd
-        )
+        skip_rewards_check = (
+            use_reference_script and build_method == clusterlib_utils.BuildMethods.BUILD_RAW
+        ) or (not use_reference_script and build_method == clusterlib_utils.BuildMethods.BUILD)
```

Comment:
Should be
```python
        skip_rewards_check = (
            use_reference_script and build_method != clusterlib_utils.BuildMethods.BUILD
        ) or (not use_reference_script and build_method == clusterlib_utils.BuildMethods.BUILD)
```
otherwise the long reward check would happen for all variants with build-estimate (with and without reference script). We are ok with having the check done just twice, once with `build-raw` and once with `build`.